connectors/src/connectors/snowflake/index.ts

```diff
@@ -80,11 +80,11 @@ export class SnowflakeConnectorManager extends BaseConnectorManager<null> {
   }
 
   async clean(): Promise<Result<undefined, Error>> {
-    throw new Error("Method clean not implemented.");
+    return new Ok(undefined);
```

Comment:
Will be implemented when we have the workflow (in progress will be done today), in the meantime we want to be able to delete it and not throw this error.